content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -201,6 +201,63 @@ them according to its `updateStrategy`.
 
 You can [perform a rolling update](/docs/tasks/manage-daemon/update-daemon-set/) on a DaemonSet.
 
+While working with the rolling updates of the DaemonSets, you will come across two important parameters: `maxSurge` and `maxUnavailable`.
+
+### maxSurge
+
+If you set this parameter to a number above 0, you allow the DaemonSet controller
+to create temporary duplicate Pods during a rollout.
+Because a DaemonSet aims to have one healthy Pod per node, this field also implies
```

Comment:
```suggestion
Because a DaemonSet aims to have one healthy Pod per schedulable node, this field also implies
```

Reply:
Or “selected”? Were you thinking of not scheduling to nodes with taints that the DaemonSet doesn't tolerate, or about the node selector for the DaemonSet's spec @atiratree ?